cmd/template/main.go

```diff
@@ -24,6 +24,8 @@ import (
 	"github.com/bank-vaults/internal/configuration"
 )
 
+var Version = "v1.31.1"
```

Comment:
i see, this has changed as well. probably better to have dev version as default and only replace on build
```suggestion
var Version = "dev"
```

Reply:
Should I change on all other PR's to dev?

Reply:
yes pleases

Reply:
So... shouldn't it be changed here as well? 🙂

Reply:
https://github.com/bank-vaults/bank-vaults/issues/2832